src/MetricsReporter.ts

```diff
@@ -70,6 +70,12 @@ export class MetricsReporter {
         this.report('engine.method_failure', point);
     }
 
+    public methodTime(method: string, time: number, tags: any = {}) {
+        const point = MetricsReporter.buildPoint(Object.assign({ method: method }, tags), {time: time});
```

Comment:
I told you originally to use the field named `time` but that's actually a reserved word in InfluxDB.  It's conflicting with the actual time value of the timeseries.  Let's change the field name we're generating in this point to `time_delta`!